python/tskit/trees.py

```diff
@@ -5073,6 +5076,71 @@ def alignments(self, *, reference_sequence=None, missing_data_character=None):
             a[site_pos] = h
             yield a.tobytes().decode("ascii")
 
+    @property
+    def individuals_population(self):
+        """
+        Returns the length-``num_individuals`` array containing, for each
+        individual, the ``population`` attribute of their nodes, or
+        ``tskit.NULL`` for individuals with no nodes. Errors if any individual
+        has nodes with inconsistent non-NULL populations.
+        """
+        if self._individuals_population is None:
+            self._individuals_population = (
+                self._ll_tree_sequence.get_individuals_population()
+            )
+        return self._individuals_population
+
+    @property
+    def individual_populations(self):
+        # Undocumented alias for individuals_population to avoid breaking
+        # pre-1.0 pyslim code
+        return self.individuals_population
+
+    @property
+    def individuals_time(self):
+        """
+        Returns the length-``num_individuals`` array containing, for each
+        individual, the ``time`` attribute of their nodes or ``np.nan`` for
+        individuals with no nodes. Errors if any individual has nodes with
+        inconsistent times.
+        """
+        if self._individuals_time is None:
+            self._individuals_time = self._ll_tree_sequence.get_individuals_time()
+        return self._individuals_time
+
+    @property
+    def individual_times(self):
+        # Undocumented alias for individuals_time to avoid breaking
+        # pre-1.0 pyslim code
+        return self.individuals_time
```

Comment:
Can we cover this line (and the populations one) if pyslim is depending on it?

Reply:
Uh-oh, I did put in something that covers that but looks like I was too quick with the search-and-replace. =)